table/block_based/block_based_table_reader_test.cc

```diff
@@ -58,14 +67,22 @@ class BlockBasedTableReaderBaseTest : public testing::Test {
         } else {
           v = rnd.RandomString(256);
         }
+        std::string user_key = std::string(k);
         if (ts_sz > 0) {
-          std::string user_key;
-          AppendKeyWithMinTimestamp(&user_key, std::string(k), ts_sz);
-          kv[user_key] = v;
+          if (same_key_diff_ts) {
+            PutFixed64(&user_key, current_udt);
+            current_udt -= 1;
+          } else {
+            PutFixed64(&user_key, 0);
+          }
+        }
+        InternalKey internal_key(user_key, seq_no, ValueType::kTypeValue);
+        kv.push_back(std::make_tuple(internal_key.Encode().ToString(), v));
```

Comment:
How about 

`kv.emplace_back(internal_key.Encode().ToString(), v);`

?

Reply:
Thank you for the suggestion! Done.